### PR TITLE
Enhancement: added automatic .meta.js geration through github actions and updated updateURL's

### PR DESCRIPTION
**Pull request by touhidurrr**
using .meta.js reduce bandwidth when checking updates. This edit will automatically genarate .meta.js file in the meta directory

### CODE REVIEW

**Comment by touhidurrr** [comment]
If you are wondering what this commit is all about please check the meta folder and the // @updateURL portion of the commit.
I forgot to update versions as it is too problematic.
If you are using dependabot, please disable it for the meta directory if you choose to merge this.

**Comment by touhidurrr** [comment]
Ok I updated the version numbers also

**Comment by touhidurrr** [comment]
Just deleted a file that I created by mistake.

File: meta/gist-raw-links.meta.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// @grant       GM_xmlhttpRequest
+// @grant       GM.xmlHttpRequest
+// @connect     api.github.com
+// @connect     assets-cdn.github.com
+// @require     https://greasemonkey.github.io/gm4-polyfill/gm4-polyfill.js?updated=20180103
+// @icon        https://github.githubassets.com/pinned-octocat.svg
+// @updateURL   https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/meta/gist-raw-links.meta.js
+// @downloadURL https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/gist-raw-links.user.js
+// @supportURL  https://github.com/Mottie/GitHub-userscripts/issues
+// ==/UserScript==
```
Comment (cmalard) [inline meta/gist-raw-links.meta.js:21]:
```diff
// ==UserScript==
-// @name        Gist Raw Links
// @version     0.2.3
-// @description Add a button that contains a list of gist raw file links
-// @license     MIT
-// @author      Rob Garrison
-// @namespace   https://github.com/Mottie
-// @include     https://gist.github.com/*
-// @run-at      document-idle
-// @grant       GM_addStyle
-// @grant       GM.addStyle
-// @grant       GM_xmlhttpRequest
-// @grant       GM.xmlHttpRequest
-// @connect     api.github.com
-// @connect     assets-cdn.github.com
-// @require     https://greasemonkey.github.io/gm4-polyfill/gm4-polyfill.js?updated=20180103
-// @icon        https://github.githubassets.com/pinned-octocat.svg
-// @updateURL   https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/meta/gist-raw-links.meta.js
-// @downloadURL https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/gist-raw-links.user.js
-// @supportURL  https://github.com/Mottie/GitHub-userscripts/issues
// ==/UserScript==
```
Considering the scope of this PR, would that work? :-)

Comment (touhidurrr) [reply meta/gist-raw-links.meta.js:21]: I think `updateURL` 	and `downloadURL` might be required depending on how it was implemented on the client’s.
Others are not required probably.

Comment (touhidurrr) [reply meta/gist-raw-links.meta.js:21]: I think I should pr again with the files.

**Pull request closed without merging**